src/NzbDrone.Core/HealthCheck/Checks/SlackUrlCheck.cs

```diff
@@ -0,0 +1,33 @@
+using System.Linq;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Localization;
+using NzbDrone.Core.Notifications;
+using NzbDrone.Core.Notifications.Slack;
+using NzbDrone.Core.ThingiProvider.Events;
+
+namespace NzbDrone.Core.HealthCheck.Checks
+{
+    [CheckOn(typeof(ProviderUpdatedEvent<INotificationFactory>))]
+    [CheckOn(typeof(ProviderDeletedEvent<INotificationFactory>))]
+    [CheckOn(typeof(ProviderStatusChangedEvent<INotificationFactory>))]
+    public class SlackUrlCheck : HealthCheckBase
+    {
+        private readonly INotificationFactory _notificationFactory;
+
+        public SlackUrlCheck(INotificationFactory notificationFactory, ILocalizationService localizationService)
+            : base(localizationService)
+        {
+            _notificationFactory = notificationFactory;
+        }
+
+        public override HealthCheck Check()
+        {
+            if (_notificationFactory.GetAvailableProviders().Where(n => n.ConfigContract.Equals("SlackSettings")).Any(s => (s.Definition.Settings as SlackSettings).WebHookUrl.Contains("discord")))
+            {
+                return new HealthCheck(GetType(), HealthCheckResult.Warning, _localizationService.GetLocalizedString("DiscordUrlInSlackNotification"));
```

Comment:
needs a fragment (and wiki update)

Reply:
go with `discord-as-slack-notification`

Reply:
Wiki text sound good?

> - You have Discord configured to use a Slack webhook. This is not advised and the native Discord functionality should be used instead.  If you're receiving this that likely means you just configured this by following an outdated guide. Please advise the guide author to update their steps. If you're receiving this after an update, please visit our support channels as it appears the automatic migration did not complete as expected.

Reply:
`If you're receiving this then it likely means you just configured this by following an outdated guide.`

I would also remove the bit about receiving this after an update as the only way people will get this is if they add it after the migration. The migration covers anything with "discord" in the webhookUrl.